services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/utils/VpcManagementUtil.java

```diff
@@ -113,6 +116,12 @@ public static VpcEntity configureNetworkDefaultParameters (VpcEntity response) {
             response.setStatus(NetworkStatusEnum.ACTIVE.getNetworkStatus());
         }
 
+        // mtu
```

Comment:
Add //TDDO in the code to remind ourselves and community that the current logic will be updated and the plan